packages/driver/cypress/e2e/commands/querying/querying.cy.js

```diff
@@ -1744,54 +1735,14 @@ space
       it('throws when assertion is have.length > 1', function (done) {
         cy.on('fail', (err) => {
           assertLogLength(this.logs, 2)
-          expect(err.message).to.eq('`cy.contains()` cannot be passed a `length` option because it will only ever return 1 element.')
+          expect(err.message).to.eq('`cy.contains()` only ever returns one element, so you cannot assert on a `length` greater than one.')
           expect(err.docsUrl).to.eq('https://on.cypress.io/contains')
 
           done()
         })
 
         cy.contains('Nested Find').should('have.length', 2)
       })
-
-      it('restores contains even when cy.get fails', (done) => {
```

Comment:
The "restores contains" tests are no longer relevant, because we're not overriding jQuery's builtin `:contains` in the first place. Nothing to clean up! :)